test/utils.ts

```diff
@@ -1,11 +1,9 @@
 import RelayHubConfiguration from '../types/RelayHubConfiguration';
 
-export function bytes32(n: number): string {
+export function createBytes32FromNumber(n: number): string {
     return '0x' + n.toString().repeat(64).slice(0, 64);
 }
```

Comment:
Sorry if it's knit-picking. I think it has to be clear though, that the generated string is not a bytes32 representation of the given number.
```suggestion
export function generateBytes32(seed: number): string {
    return '0x' + seed.toString().repeat(64).slice(0, 64);
}
```